selfdrive/car/hyundai/values.py

```diff
@@ -112,6 +112,7 @@ class CAR(StrEnum):
   KIA_NIRO_EV = "KIA NIRO EV 2020"
   KIA_NIRO_EV_2ND_GEN = "KIA NIRO EV 2ND GEN"
   KIA_NIRO_PHEV = "KIA NIRO HYBRID 2019"
+  KIA_NIRO_PHEV_2022 = "KIA NIRO PLUG-IN HYBRID 2022"
```

Comment:
Can we combine with `KIA_NIRO_HEV_2021`?

Reply:
It needs to be separated as this car would fault if we send the signals from this block under `hyundaican.py`:
https://github.com/commaai/openpilot/blob/bc70c94f75f64c31998b9796f20ec6e310cae347/selfdrive/car/hyundai/hyundaican.py#L36-L57